src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -189,6 +189,49 @@ void braid_init_example_state(Node &res)
 }
 
 
+//---------------------------------------------------------------------------//
+inline float64 braid_init_example_point_scalar_field_calc_dx(index_t npts_x)
+{
+    return (float) (4.0 * PI_VALUE) / float64(npts_x - 1);
+}
+
+
+//---------------------------------------------------------------------------//
+inline float64 braid_init_example_point_scalar_field_calc_dy(index_t npts_y)
+{
+    return (float) (2.0 * PI_VALUE) / float64(npts_y-1);
+}
+
+
+//---------------------------------------------------------------------------//
+inline float64 braid_init_example_point_scalar_field_calc_dz(index_t npts_z)
+{
+    return (float) (3.0 * PI_VALUE) / float64(npts_z-1);
+}
```

Comment:
How do we feel about these? I did this instead of hard coding the calculation wherever it was needed.

Reply:
The most general way would be to eval at a specific x,y,z instead of the dx,dy,dz. 
Should be an easy refactor, what do you think?

Reply:
Why did I do this? If I remember correctly, it was because I didn't want to recalculate dx, dy, and dz for each x,y,z I am using to calculate the field val. There is a function (below, probably?) that calculates the field val for a given x,y,z and dx,dy,dz. Does that make sense?